drivetrain.py

```diff
@@ -24,15 +30,21 @@ def __init__(self, timeout=120):
         self.start_time = time.clock()
         self.pz = piconzero
         self.pz.init()
+        time.sleep(0.5)
         self.motor_max = 100
+        self.pz.setInputConfig(BATT_ADC_PIN, 1)
         self.slow_speed = 20
         self.deadband = 1
         self.boost_cycles = 1
         self.boost_dwell = 9
         self.name = "Drivetrain"
+
+        # value to set if motor speed should be held consistent across battery discharge
+        self.NORMALISE_MOTOR_SPEED = True

```

Comment:
suggest a more appropriate name would be `should_normalise_motor_speed`, in lower case.